lib/node_modules/@stdlib/blas/base/zdotu/test/test.zdotu.js

```diff
@@ -0,0 +1,217 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var tape = require( 'tape' );
+var Complex128 = require( '@stdlib/complex/float64/ctor' );
+var Complex128Array = require( '@stdlib/array/complex128' );
+var isSameComplex128 = require( '@stdlib/assert/is-same-complex128' );
+var zdotu = require( './../lib/zdotu.js' );
+
+
+// TESTS //
+
+tape( 'main export is a function', function test( t ) {
+	t.ok( true, __filename );
+	t.strictEqual( typeof zdotu, 'function', 'main export is a function' );
+	t.end();
+});
+
+tape( 'the function has an arity of 5', function test( t ) {
+	t.strictEqual( zdotu.length, 5, 'returns expected value' );
+	t.end();
+});
+
+tape( 'the function calculates the dot product of two double-precision complex floating-point vectors `x` and `y`', function test( t ) {
+	var dot;
+	var x;
+	var y;
+
+	x = new Complex128Array( [ 4.0, 2.0, -3.0, 5.0, -1.0, 2.0, -5.0, 6.0 ] );
+	y = new Complex128Array( [ 2.0, 6.0, -1.0, -4.0, 8.0, 8.0, 2.0, -3.0 ] );
+
+	dot = zdotu( 4, x, 1, y, 1 );
+	t.strictEqual( isSameComplex128( dot, new Complex128( 3, 70 ) ), true, 'returns expected value' );
+
+	x = new Complex128Array( [ 3.0, -4.0, 1.0, 2.0 ] );
+	y = new Complex128Array( [ 1.0, -2.0, 3.0, 8.0 ] );
+
+	dot = zdotu( 2, x, 1, y, 1 );
+	t.strictEqual( isSameComplex128( dot, new Complex128( -18, 4 ) ), true, 'returns expected value' );
+
+	t.end();
+});
+
+tape( 'if provided an `N` parameter less than or equal to `0`, the function returns `0 + 0i`', function test( t ) {
```

Comment:
```suggestion
tape( 'if provided an `N` parameter less than or equal to `0`, the function returns `0`', function test( t ) {
```

Consistency with other test file.